modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -68,59 +67,69 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 +
 // Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
 
-. Convert the `journal.conf` file to base64:
+. Convert the `journal.conf` file to base64 and store it in a variable (`jrnl_cnf`) by running the following command:
 +
 [source,terminal]
 ----
-$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+$ export jrnl_cnf=$( cat journald.conf | base64 -w0 )
 ----
 
-. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step.
-+
-For example:
+. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which was created in the previous step. The following sample command creates a MachineConfig for the  worker:
 +
 [source,terminal]
 ----
-$ cat > /tmp/40-worker-custom-journald.yaml <<EOF
+$ cat << EOF > ./99-worker-journald-configuration.yaml <1>
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 50-corp-journald
+    machineconfiguration.openshift.io/role: worker <2>
+  name: 99-worker-journald-configuration <3>
 spec:
   config:
     ignition:
-      version: 3.1.0
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 3.2.0
+    networkd: {}
+    passwd: {}
     storage:
       files:
       - contents:
-          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf}
-        mode: 0644 <1>
-        overwrite: true
-        path: /etc/systemd/journald.conf <2>
+          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf} <4>
+          verification: {}
+        filesystem: root
+        mode: 0644 <5>
+        path: /etc/systemd/journald.conf.d/custom.conf
+  osImageURL: ""
 EOF
 ----
-<1> Set the permissions for the `journal.conf` file. It is recommended to set `0644` permissions.
-<2> Specify the path to the base64-encoded `journal.conf` file.
+<1> Optional: For control plane (also known as master) node, you can provide the file name as `00-master-journald-configuration.yaml`.
```

Comment:
Apologies for not noticing this earlier and I am unsure if it has any consequence, but why `99-worker...` for the worker and `00-ma...` for the control plane?  Is there a reason that it would not also be `99-master-...`?

Reply:
Based on my understanding, the change of names suggested by reporter was just a part of enhancement and wasn't tied to any technical constraints. I pasted the name suggested by reporter and I do not think there should be any consequence. But there could be some technical reason behind the readability of nomenclature.